Correção controladores

DankoCaboski opened the pull request:
Adicionados os controladores das telas de home do colaborador, gestor e do administrador.

DankoCaboski committed:
Correção controladores

Adicionados os controladores das telas de home do colaborador, gestor e do administrador.

larissa-fernanda reviewed (changes requested):
Resolva os conflitos 😸

DankoCaboski committed:
FXML colaborador

Correções

DankoCaboski committed:
Correção de conflitos

Corrigidos os conflitos da branch

DankoCaboski committed:
Correção de conflito

DankoCaboski committed:
Correção de conflito

DankoCaboski committed:
.

DankoCaboski committed:
Classe "Usuario" e "Conector"

Criada a classe "Usuario" para armazenar as informações dos usuários .
Criado  pacote "Conexao" retrabalhar a classe Conector pois essa não está  funcionando.

DankoCaboski commented:
Correções realizadas

larissa-fernanda committed:
Merge branch 'main' into NavegacaoDeTelas

NicSouza committed:
Tela de aprovação do Gestor

NicSouza committed:
Merge pull request #6 from codecatss/TelasGestor

Telas gestor
